resources/home/dnanexus/tests/make_expected_test_beds.py

```
@@ -0,0 +1,59 @@
+    ).astype({"chromosome": str, "start": int ,"end": int, "transcript": str})
+
+       
+    expected_bed_file.to_csv(f"{test_data_dir}/expected_beds/test_expected_bed_file_with_additional_regions.bed", sep="\t", header=False, index=False)
+
+TEST_DATA_DIR=(
+    os.path.join(os.path.dirname(__file__), 'test_data')
+)
+
+expected_bed_file(test_data_dir=TEST_DATA_DIR)
+expected_bed_file_with_flank(test_data_dir=TEST_DATA_DIR)
+make_expected_bed_file_with_addtional_regions(test_data_dir=TEST_DATA_DIR)
```

Ruff (0.15.12)

[error] Undefined name `make_expected_bed_file_with_addtional_regions`

(F821)